recipes/PyFstat/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "PyFstat" %}
+{% set version = "1.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5275804036a0d814acf111857afd46fbb3cba81e48f3d5275fbdd7219d75dddc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >3.6,<3.9
+    - pip
+  run:
+    - python
+    - numpy
+    - matplotlib >=2.1
+    - scipy
+    - ptemcee
+    - corner
+    - dill
+    - tqdm
+    - bashplotlib
+    - peakutils
+    - pathos
+    - lalsuite >=6.76
+    - versioneer
+
+test:
+  requires:
+    - pytest
+  imports:
+    - unittest
+    - numpy
+    - os
+    - shutil
+    - pyfstat
+    - lalpulsar
+    - logging
+    - time
+    - scipy
```

Most of these are in the standard library or already in `run`. I would probably suggest adding the remaining two to `run` as well:
```suggestion
    - pyfstat
    - lalpulsar
```

---

Ah ok, I was following the comment in the example recipe to `List all the packages/modules that `run_test.py` imports.` and didn't realize they can be trimmed down if overlapping with `run`.

`lalpulsar` is actually provided by the `lalsuite` dependency under `run` as well, so I guess I can trim this down to just the package itself (`pyfstat`)?

---

Changed to just `- pyfstat` now.